pkg/config/api_config.go

```diff
@@ -67,64 +80,116 @@ func LoadAPIConfigFromFile(path string) (*fc.APIConfig, error) {
 
 // LoadAPIConfig load the api config from config center
 func LoadAPIConfig(metaConfig *model.APIMetaConfig) (*fc.APIConfig, error) {
-	client = etcdv3.NewConfigClient(
+	client, _ = etcdv3.NewConfigClientWithErr(
```

Comment:
should we handle the error return from the new client function?

Reply:
It also limited the config center can only be etcd. Could we get the config center client from something like getConfigCenterClient(some_indicator)? It is not generic enough I think.

Reply:
already handle the error return from the new client function